src/javascript/Builders/TextBuilder.js

```diff
@@ -0,0 +1,53 @@
+import * as THREE from "three";
+import Transforms from "../Utils/Transforms";
+import MaterialBuilder from "./MaterialBuilder";
+import { FontLoader } from 'three/examples/jsm/loaders/FontLoader.js';
+import { TextGeometry } from 'three/examples/jsm/geometries/TextGeometry';
+
+class TextBuilder {
+  static BuildText(options, parent) {
+
+    const loader = new FontLoader();
+    
+    loader.load(
```

Comment:
Do we really want to load each font for each TextGeometry? 

Would it possibly more efficient to store each the unique fonts in memory in the case where multiple TextGeometries need to be constructed? This way the method could reference a font that has already been loaded or load it just in time if we haven't already.